tests/e2e/suite_test.go

```diff
@@ -48,7 +48,7 @@ var (
 	ctx           context.Context
 	cancel        context.CancelFunc
 	suiteTimeout  = 5 * time.Minute
-	testTimeout   = 1 * time.Minute
+	testTimeout   = 2 * time.Minute
```

Comment:
may need a little longer since we now need to wait for longer things like ISBService to be fully created, etc